pegasus/sites.v3/code.org/public/pd-workshop-survey/splat.haml

```diff
@@ -189,6 +189,9 @@ title: "Professional Development Workshop Survey"
           name: 'how_motivating_s',
           label: 'During your workshop, how motivating were the activities that this program had you do?'
 
+      - facilitators.each do |name|
+        %input{name: 'facilitator_names_s[]', type: 'hidden', value: name}
```

Comment:
Yep! Multiple inputs with the same name will automatically be serialized to an array of values